config/ember-try.js

```diff
@@ -74,6 +75,8 @@ module.exports = async function () {
           },
         },
       },
+      embroiderSafe(),
```

Comment:
```suggestion
      embroiderSafe({allowedToFail: true}),
```
something like this

Reply:
see example here https://github.com/storybookjs/ember-cli-storybook/blob/27bf1b433ba86c88fdf8f858b235ebaac6355035/config/ember-try.js#L86

Reply:
Thanks! I was trying to get the `API` and check the existing options, seems to be good